ansible_collections/arista/avd/plugins/action/anta_catalog_builder.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from typing import Any, Dict, List, Tuple
+
+from ansible.plugins.action import ActionBase
+from anta.loader import parse_catalog
+
+TestTuple = Tuple[str, Dict[str, str]]
+Catalog = Dict[str, List[Dict[str, Any]]]
+
+
+AVD_MAPPING: Dict[str, List[TestTuple]] = {
```

Comment:
This is where we map Ansible task_vars to ANTA different test cases, including the right parameters as inputs. Another mapping example would be if `banners` key exists in task_vars, run the relevant banner test cases of ANTA with `banners.motd` and `banners.login` as inputs.

This mapping could get quite large the more tests requiring inputs we add in ANTA. It is also a manual process, meaning if we add a new test in ANTA, we need to update the mapping if needed.

Reply:
Catalog handling has been merged in `anta_runner.py`.